common/interfaces/desktop_app_interface/usb_internals.c

```diff
@@ -407,6 +414,10 @@ static void comm_write_packet(const uint16_t chunk_number,
 /*****************************************************************************
  * GLOBAL FUNCTIONS
  *****************************************************************************/
+void comm_reset_interface(void) {
```

Comment:
Can we move this function to usb_api.c?

Reply:
I've defined this function in the file where the variable is also declared. What's the motivation to move this?

Reply:
Motivation is simply to keep all APIs in api file and not internal file